packages/react-composites/src/composites/CallWithChatComposite/CallWithChatPane.tsx

```diff
@@ -182,7 +186,9 @@ const sidePaneStyles: IStackStyles = {
   }
 };
 
-const availableSpaceStyles: IStackStyles = { root: { width: '100%', height: '100%' } };
+const availableSpaceStyles = (theme: Theme): IStackStyles => ({
+  root: { width: '100%', height: '100%', position: 'absolute', zIndex: 1, background: theme.palette.white }
```

Comment:
Any suggestions for handling the background color?
In any case, we will have to specify a color to the chat pane Or it will be transparent

Reply:
my thinking was applying it to the root of the callwithchat or chat composite, but it doesn't really make a difference (and thinking more it would be wrong to apply to the root of the chat composite as it might want to be transparent) so fine to leave here!